src/style.css

```diff
@@ -1,3 +1,24 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@100;200;300;400;500;600;700&family=Ubuntu:wght@300;400;500;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Ubuntu", sans-serif;
+}
+
+/* body {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  gap: 50px;
+}
+
+#title {
+  text-align: left;
+}
```

Comment:
## Recommendation Change

- [ ] I noticed that you used Bootstrap for your styles. Your app is nicely styled and so I recommend you remove all the commented styles in your CSS file that you're not using.

Reply:
Fixed.